crates/chain/tests/test_local_chain.rs

```diff
@@ -32,6 +32,7 @@ enum ExpectedResult<'a> {
     Err(CannotConnectError),
 }
 
+#[allow(clippy::print_stdout)]
 impl<'a> TestLocalChain<'a> {
     fn run(mut self) {
         println!("[TestLocalChain] test: {}", self.name);
```

Comment:
🔧 I'd rather you remove this `println` than add the allow.

Reply:
Sure, I'll finish removing it in the further commits. There are a bunch of `println` used throughout the code, so would require attention and judgment of what we "might" need/want to keep.

Reply:
I did it in the new commits, let me know what you think.